hw/ip/otbn/dv/smoke/run_smoke.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Runs the OTBN smoke test (builds software, build simulation, runs simulation
+# and checks expected output)
+
+set -o pipefail
+
+if [ ! -f './util/build_consts.sh' ]; then
+  echo "Must be run from OpenTitan repository root"
+  exit 1
+fi
+
+source ./util/build_consts.sh
+
+SMOKE_BIN_DIR=$BIN_DIR/otbn/smoke_test
+SMOKE_SRC_DIR=$REPO_TOP/hw/ip/otbn/dv/smoke
+
+mkdir -p $SMOKE_BIN_DIR
+
+$REPO_TOP/hw/ip/otbn/util/build.sh $SMOKE_SRC_DIR/smoke_test.S \
+  $SMOKE_BIN_DIR/smoke
+
+if [ $? -ne 0 ]; then
+  echo "OTBN SMOKE FAILURE: Software build failed"
+  exit 1
+fi
+
+fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ip:otbn_top_sim
+
+if [ $? -ne 0 ]; then
+  echo "OTBN SMOKE FAILURE: HW Sim build failed"
+  exit 1
+fi
+
+RUN_LOG=`mktemp`
```

Comment:
We're not cleaning up this temporary file. You'll need to install a trap handler to do so.